server.js

```diff
@@ -20,8 +21,81 @@ app.use(cors());
 app.use(express.json());
 
 // Start defining your routes here
+
+// Route to return entire song array
+app.get("/songs", (req, res) => {
+  res.json(topMusicData);
+});
+
+// Route for songs by artist
+app.get("/songs/artist/:artistName", (req, res) => {
+  const artistName = req.params.artistName.toLowerCase();
+  const songsByArtist = topMusicData.filter(
+    (song) => song.artistName.toLowerCase() === artistName
+  );
+
+  if (songsByArtist.length > 0) {
+    res.json(songsByArtist);
+  } else {
+    res.status(404).json({ error: "No songs found for this artist" });
+  }
+});
```

Comment:
This could be a query param under the songs endpoint to make it more RESTful: `/songs?artist=beyonce`